src/seedu/addressbook/AddressBook.java

```diff
@@ -508,7 +526,7 @@ private static String executeDeletePerson(String commandArgs) {
             return MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
         }
         final String[] targetInModel = getPersonByLastVisibleIndex(targetVisibleIndex);
-        return deletePersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
+        return hasDeletedPersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
```

Comment:
Good change. The method name is now quite confusing to reader. It is no longer clear whether the method searches ("hasXXX") or perform deletion.